### PR TITLE
Changes for dropdown crash and UI fix for ios

### PR DESCRIPTION
**Pull request by Sofiya-kumar**
Changes for dropdown crash while closing the popup.

### CODE REVIEW

**Comment by Sofiya-kumar** [comment]
@rthanga1 
Below are points addressed in this PR. 

1) Dropdown Crash
2) Added ControlLabel control to use the same as Input label
3) Added that Control Label In Dropdown
4) Switch Text binding is not working. So fixed that
5) Switch position left or right is required so added that option as well
6) Helper Text is not wrapping in TmInput

File: Trimble.Modus.Components/Controls/Label/ControlLabel.xaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+             x:Name="controlLabelView">
+  <ContentView.Resources>
+    <ResourceDictionary>
+      <toolkit:IsStringNotNullOrEmptyConverter x:Key="IsStringNotNullOrEmptyConverter" />
+      <ResourceDictionary.MergedDictionaries>
+        <local:ControlLabelStyles />
+      </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+  </ContentView.Resources>
+  <Grid Grid.Row="0"
```
Comment (jsramraj) [inline Trimble.Modus.Components/Controls/Label/ControlLabel.xaml:16]: Any reason to use Grid here? Can't you just the simple StackLayout?

File: Trimble.Modus.Components/Controls/Label/ControlLabel.xaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+           Grid.Column="0"
+           FontFamily="OpenSansSemibold"
+           Text="{Binding TitleText}"
+           VerticalOptions="Center"
+           HorizontalOptions="Start" />
+    <Label Text="*"
+           Grid.Column="1"
+           IsVisible="{Binding IsRequired}"
+           FontFamily="OpenSansSemibold"
+           TextColor="#DA212C" />
```
Comment (jsramraj) [inline Trimble.Modus.Components/Controls/Label/ControlLabel.xaml:33]: Hardcoded color? Can we use named colors? Can you check in the input?

File: Trimble.Modus.Components/Enums/SwitchLabelPosition.cs [hunk 3]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Trimble.Modus.Components.Enums;
+
+public enum SwitchLabelPosition
```
Comment (jsramraj) [inline Trimble.Modus.Components/Enums/SwitchLabelPosition.cs:3]: Can you make it generic? Like Label or TitlePosition? We can use it in other controls too in the future if necessary

File: Trimble.Modus.Components/Controls/Label/ControlLabel.xaml (hunk 1 above)
Comment (Sofiya-kumar) [reply Trimble.Modus.Components/Controls/Label/ControlLabel.xaml:16]: Nope. But in some cases stack layout has some UI alignment issues in other projects. so used grid here

File: Trimble.Modus.Components/Enums/SwitchLabelPosition.cs (hunk 3 above)
Comment (Sofiya-kumar) [reply Trimble.Modus.Components/Enums/SwitchLabelPosition.cs:3]: Sure

**Review by jsramraj — APPROVED** [review]

File: Trimble.Modus.Components/Controls/Label/ControlLabel.xaml (hunk 2 above)
Comment (Sofiya-kumar) [reply Trimble.Modus.Components/Controls/Label/ControlLabel.xaml]: Modified with static resources colors

**Review by rthanga1 — APPROVED** [review]

**Pull request merged**